posttest.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('./test/init.js');
+
+var async = require('async');
+var _ = require('lodash');
+
+var db = global.getDataSource();
+var schemaName = global.config.schema;
+var findTables = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'T\'' + ';';
+var findViews = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'V\'' + ';';
+
+if (!process.env.CI)
+  return console.log('Skipping post-test ...');
+console.log('> Cleaning up the database ...');
+
+// drop all the tables and views from the schema
+db.connector.execute(findViews, function(err, views) {
+  if (err) throw err;
+
+  async.each(views, function(view, callback) {
+    var viewName = view.TABNAME;
+    db.connector.execute('DROP VIEW "' + schemaName + '"."' +
+      viewName + '";', function(err, result) {
+      callback(err);
+    });
+  }, function(err) {
+    if (err) throw err;
+
+    db.connector.execute(findTables, function(err, tables) {
+      if (err) throw err;
+
+      async.each(tables, function(table, callback) {
+        var tableName = table.TABNAME;
+        db.connector.execute('DROP TABLE "' + schemaName + '"."' +
+          tableName + '";', function(err, result) {
+          callback(err);
+        });
+      }, function(err) {
+        if (err) throw err;
+        db.connector.execute('DROP SCHEMA "' + schemaName +
```

Comment:
I wondered the same as well. But dashdb (at least the version we have on the container) only supports restricted version, i.e need to drop all the objects on the schema before dropping the schema itself. :(